functional_tests/keras/keras_nonnumeric.yea

```diff
@@ -0,0 +1,17 @@
+id: 0.keras.nonnumericlayers
+tag:
+    shard: tf25
+plugin:
+    - wandb
+command:
+    program: test_keras_nonnumeric.py
+depend:
+    requirements:
+        - tensorflow==2.5.0
```

Comment:
is this feature only works for tf=2.5?

Reply:
Prior to 2.5 I can't find any experimental/ or otherwise layers that have non-numeric values like a StringLookup preprocessing layer. I've modified the test so that it tests the string values in both 2.5 and 2.6